client/core/core.go

```diff
@@ -4506,12 +4885,10 @@ func (c *Core) connectDEX(acctInfo *db.AccountInfo, temporary ...bool) (*dexConn
 		wsCfg.NetDialContext = proxy.DialContext
 	}
 
-	if listen {
-		wsCfg.ReconnectSync = func() {
-			go c.handleReconnect(host)
-		}
-		wsCfg.ConnectEventFunc = dc.handleConnectEvent
+	wsCfg.ReconnectSync = func() {
+		go c.handleReconnect(host)
 	}
+	wsCfg.ConnectEventFunc = dc.handleConnectEvent
```

Comment:
This method's comment says `If temporary is provided and true, no connect and reconnect handlers are registered`. May need to take off that bit and add a comment here with the rationale for allowing reconnections on temp dcs.